api/v1alpha4/conversion.go

```diff
@@ -612,8 +616,57 @@ func (dst *NnfSystemStorageList) ConvertFrom(srcRaw conversion.Hub) error {
 	return apierrors.NewMethodNotSupported(resource("NnfSystemStorageList"), "ConvertFrom")
 }
 
+// DWS ResourceError conversion routines.
```

Comment:
This section of `Convert` and `autoConvert` routines for the DWS ResourceError is what conversion-gen needed to see. I copied them out of the DWS v1alpha2 `zz_generated.conversion.go` and massaged them just enough to compile here.